browser/ui/views/tabs/brave_tab_context_menu_contents.cc

```diff
@@ -245,8 +244,7 @@ void BraveTabContextMenuContents::ExecuteBraveCommand(int command_id) {
       SwapTabsInTile();
       return;
     default:
-      NOTREACHED_IN_MIGRATION();
-      return;
+      NOTREACHED() << "All commands are handled above";
```

Comment:
Whenever possible, just move `NOTREACHED` outside the switch, and delete the `default:` entry.